src/wizards/tokenSwapDealWizard/stages/submitStage/submitStage.html

```diff
@@ -111,7 +111,7 @@ <h4 class="submitCardHeader">Primary DAO</h4>
           </section>
 
           <section class="stageSectionContent">
-            <pcard if.bind="submitData.partnerDAO">
+            <pcard if.bind="!isOpenProposalLike">
```

Comment:
Why this change?
It makes sense, but I would prefer the previous one, as the section is about "Partner DAO", and the `if.bind="submitData.partnerDAO"` is a bit more descriptive

Reply:
If we use `submitData.partnerDAO` here in the template it adds `partnerDAO` property to `submitData` with value `undefined`. We don't want to have `partnerDAO` property at all

Reply:
I really doubt that just referencing a property in if.bind would cause the property to be altered.

Reply:
I was shocked but it did create the property. According to my debugging, referencing a non-existing property in if.bind, adds that property to the object with value undefined

Reply:
Maybe that is so it has something to observe.  I never realized that was how it did it.

Reply:
I can see this being a potenial bug, but I couldn't reproduce this "adds non-existing prop" behavior

STEPS
1. readd if.bind="submitData.partnerDAO"
2. in bind() hook add `console.log(JSON.stringify(submitData, null, 4));
3. in attached() hook add the same `console.log(JSON.stringify(submitData, null, 4));
RESULT
4. no `partnerDAO` prop was added